units/commit_queue.go

```diff
@@ -187,12 +198,20 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		}
 
 		grip.Info(message.Fields{
-			"source":  "commit queue",
-			"job_id":  j.ID(),
-			"item":    nextItem,
-			"message": "finished processing item",
+			"source":               "commit queue",
+			"job_id":               j.ID(),
+			"item":                 nextItem,
+			"message":              "finished processing commit queue item",
+			"processing_time_secs": time.Since(beginProcessingTime).Seconds(),
 		})
 	}
+	grip.Info(message.Fields{
+		"source":               "commit queue",
+		"job_id":               j.ID(),
+		"items":                nextItems,
```

Comment:
Is items needed to log? I'd rather we didn't since we log this so frequently. Is the use of this the length of items?

Reply:
It's not explicitly needed to log, I just thought it'd be helpful to give more visibility into what's being processed. I'm fine to change this to showing the length, but I think we should log the full item at least once.